tensorflow/python/ops/math_ops.py

```diff
@@ -2795,6 +2795,23 @@ def _calc_mat_mul_flops(graph, node):
   return ops.OpStats("flops", (k * output_count * 2))
 
 
+@ops.RegisterStatistics("BatchMatMul", "flops")
+def _calc_mat_mul_flops(graph, node):
```

Comment:
function already defined line 2784 [function-redefined], please fix it

Reply:
Hi @ry, I just renamed the function name to fix the function-redefined problem